service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/base/SharedNotifyContentProvider.java

```diff
@@ -39,11 +39,9 @@ protected SharedNotifyContentProvider(String uiBaseUrl, ObjectMapper mapper) {
             .map(Hearing::getTimeFrame)
             .filter(StringUtils::isNotBlank);
 
-        // What if we just add the list no matter if it is empty or not,
-        // surely the parts below determine if it is going to be shown or not
-        template.setOrdersAndDirections(!ordersAndDirections.isEmpty() ? ordersAndDirections : List.of(""));
-        template.setDataPresent(!ordersAndDirections.isEmpty() ? YES.getValue() : NO.getValue());
-        template.setFullStop(!ordersAndDirections.isEmpty() ? NO.getValue() : YES.getValue());
+        template.setOrdersAndDirections(ordersAndDirections);
+        template.setDataPresent(YES.getValue());
+        template.setFullStop(NO.getValue());
```

Comment:
Maybe not now but should we create a ticket to update the notification + this code to remove these redundant fields? Seeing as it just appears to be constant now there isn't much point in having extra conditional logic in the notification template.

Reply:
Also if they are always going to be Yes and No respectively, we could just set that directly in the model.

Reply:
Agreed but I also think we should wait until https://tools.hmcts.net/jira/browse/FPLA-1513 has been played also as it will likely result in notify template changes also.